source/mvgApi.js

```diff
@@ -40,8 +42,57 @@ function getDepartures(station, options, apiUrl = defaultApi) {
             }
         });
     });
+}
+
+/**
+ *
+ * @param suggestion
+ * @param options
+ * @param apiUrl
+ * @returns {Promise}
+ * @TODO: add reject and error handling
+ */
+function getDeparturesBySuggestion(suggestion, options, apiUrl = defaultApi) {
+
+    return new Promise((resolve, reject) => {
+        getSuggestions(suggestion, apiUrl)
+            .then(function (suggestions) {
+                return getDepartures(suggestions[0].getStationName(), options, apiUrl)
+            })
+            .then(function (lines) {
+                resolve(lines)
+            });
+    });
+
+}
+
+/**
+ *
+ * @param name
+ * @param apiUrl
+ * @returns {Promise}
+ * @TODO: add reject and error handling
```

Comment:
Same as above.

Reply:
to be fixed next few days (got some serious workload from studies ;-) )